lib/bloom.js

```diff
@@ -69,9 +71,10 @@ Bloom.prototype.check = function (e) {
 }
 
 /**
- * checks if multple topics are in a bloom
- * @method check
- * @param {Buffer} element
+ * checks if multiple topics are in a bloom
+ * @method multiCheck
+ * @param {Buffer} topics
+ * @returns {boolean} Returns {@code true}
```

Comment:
Not sure how much consistency we have with this in the code in general, but we should see that we stay with one convention using capital letter at the explanatory comment ("Returns...") or not.

Reply:
I had noticed that a majority of the `@returns` clauses started with "Returns" so I'm following that—I can update the comments for consistency in a separate PR if we want?

Reply:
Ah, I think that's overblown for now. If you do an update with the other changes that should suffice for now.